src/Build.UnitTests/BackEnd/ProcessIdTask.cs

```diff
@@ -11,13 +12,16 @@ namespace Microsoft.Build.UnitTests
     /// </summary>
     public class ProcessIdTask : Task
     {
+        [Output]
+        public Int32 pid { get; set; }
```

Comment:
```suggestion
        public int Pid { get; set; }
```

style nits: use the C# type where possible, and properties should start with a capital letter.